src/MetaMorpho.sol

```diff
@@ -792,6 +792,11 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         (uint256 totalSupplyAssets, uint256 totalSupplyShares, uint256 totalBorrowAssets,) =
             MORPHO.expectedMarketBalances(marketParams);
 
+        // In case a call to the vault is done during a flashloan on Morpho Blue we can have:
+        // totalSupplyAssets - totalBorrowAssets > ERC20(marketParams.loanToken).balanceOf(address(MORPHO)
+        // To keep ERC4626 compliance we must return the available liquidity at the moment:
+        // ERC20(marketParams.loanToken).balanceOf(address(MORPHO)
+        // Hence, the min in the following statement:
```

Comment:
```suggestion
        // Inside a flashloan callback, liquidity on Morpho Blue can be limited to the singleton's balance.
```

Reply:
The can sounds weird, it seems like this is a feature